File: unidade-3/java/Exemplo Heranca/src/Principal.java
/*
 *	Disciplina: Programação Orientada a Objetos 
 *  Professor: 	Orlewilson B. Maia
 *  Turma:		CIN04S1
 *  Data:		18/10/17
 *  Autor:		Orlewilson B. Maia
 *  Descrição:	Executar o projeto
 * 	
 */
public class Principal {
	
	// executar o projeto
	public static void main(String[] args) {
		
		//Pessoa pes1 = new Pessoa();
		Aluno al = new Aluno("Lukas", "16169948", 
				'M',"26/09/1996","POO, SO", "CC",
				8.1, 4, "Presencial", true);
		
		Professor prof = new Professor("Orlewilson Maia", 
				"1231231", 'M', "05/05/1984", "POO",
				"Doutor", 3000.0, "Uninorte, FMF", 40,
				2);
		
		al.mostrarDados();
		
		prof.mostrarDados();	
		
	}
}

File: unidade-3/python/Principal.py
"""
Disciplina:	Programação Orientada a 
		    Objetos

Professor:	Orlewilson B. Maia
Data:		31/08/2017
Autor:		Orlewilson B. Maia
Descrição:	Arquivo para rodar exemplo
			da classe Caneta

Nome do arquivo: Principal.py

"""

# importar arquivo
from Jogador import Jogador
from CartaNormal import CartaNormal
from CartaEspecial import CartaEspecial

# para embaralhar lista
import random





# lista das cartas normal e especial
cartasNormal = []
cartasEspecial = []

# criando as cartas normal
for cor in ["vermelho", "azul","verde", 
"amarelo"]:
	for cont in range(0,2):
		for number in range(0, 10):
			cartasNormal.append(
				CartaNormal(cor, number))

# criando as cartas especial
cartasEspecial.append(
	CartaEspecial("vermelha", "+2"))
cartasEspecial.append(
	CartaEspecial("verde", "+2"))
cartasEspecial.append(
	CartaEspecial("amarela", "+2"))
cartasEspecial.append(
	CartaEspecial("azul", "+2"))

cartasEspecial.append(
	CartaEspecial("vermelha", "voltar"))
cartasEspecial.append(
	CartaEspecial("verde", "voltar"))
cartasEspecial.append(
	CartaEspecial("amarela", "voltar"))
cartasEspecial.append(
	CartaEspecial("azul", "voltar"))

cartasEspecial.append(
	CartaEspecial("vermelha", "bloquear"))
cartasEspecial.append(
	CartaEspecial("verde", "bloquear"))
cartasEspecial.append(
	CartaEspecial("amarela", "bloquear"))
cartasEspecial.append(
	CartaEspecial("azul", "bloquear"))

cartasEspecial.append(
	CartaEspecial("preta", "+4"))
cartasEspecial.append(
	CartaEspecial("preta", "mudarcor"))


# embaralhar lista
random.shuffle(cartasNormal)

for carta in cartasNormal:
      print (carta.getCor())
      print (carta.getNumero())




